### ISSUE TITLE
auditwheel not finding shared libraries

### ISSUE THREAD

**Issue by youngpm** [pypa/auditwheel#52]
I've been playing with manylinux builds and can't seem to get auditwheel to ever find shared libraries and fix them.

For example, I've tried the following by cloning `https://github.com/pypa/python-manylinux-demo` , navigating to its root directory and running
```
docker run -v `pwd`:/io -it --rm quay.io/pypa/manylinux1_x86_64 /bin/bash
```

From inside the container, I then build the wheels via
```
/io/travis/build-wheels.sh
```
Everything completes successfully, but the `auditwheel repair` doesn't seem to have packaged anything up and the wheels have no `.libs` directory containing atlas.  

From inside the container, running `auditwheel -vv show python_manylinux_demo-1.0-cp27-cp27mu-linux_x86_64.whl` gives the following:
```
DEBUG:auditwheel.wheel_abi:{}
DEBUG:auditwheel.policy.versioned_symbols:Required symbol versions: {}
INFO:auditwheel.wheel_abi:{
    "manylinux1_x86_64": {
        "priority": 100,
        "libs": {}
    },
    "linux_x86_64": {
        "priority": 0,
        "libs": {}
    }
}
DEBUG:auditwheel.wheel_abi:external referene info
DEBUG:auditwheel.wheel_abi:{
    "manylinux1_x86_64": {
        "priority": 100,
        "libs": {}
    },
    "linux_x86_64": {
        "priority": 0,
        "libs": {}
    }
}

python_manylinux_demo-1.0-cp27-cp27mu-linux_x86_64.whl is consistent
with the following platform tag: "manylinux1_x86_64".

The wheel references no external versioned symbols from system-
provided shared libraries.

The wheel requires no external shared libraries! :)
```
but unzipping that wheel and runnning `ldd` tells a different story
```
[root@1965501e43ea pymanylinuxdemo]# ldd extension.so
        linux-vdso.so.1 =>  (0x00007ffc52bd8000)
        libcblas.so.3 => /usr/lib64/atlas/libcblas.so.3 (0x00007fdcd467e000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007fdcd4461000)
        libc.so.6 => /lib64/libc.so.6 (0x00007fdcd4108000)
        libatlas.so.3 => /usr/lib64/atlas/libatlas.so.3 (0x00007fdcd3813000)
        libm.so.6 => /lib64/libm.so.6 (0x00007fdcd358f000)
        /lib64/ld-linux-x86-64.so.2 (0x00005619467b3000)
```
Any help would be greatly appreciated!   I'm experiencing the same thing across the board with other builds I've been trying to manylinuxize.

**Comment by daa** [comment]
pip show pyelftools

And if it's greater or equal to 0.24, I suspect it's manifestation of #49.

**Comment by njsmith** [comment]
I can confirm that the current docker image is shipping pyelftools 0.24. Oops. Just merged #49, which should lead to a fix once we sort out the logistics of getting the fix into the docker image. Thanks @daa for tracking this down and staying on top of it.

As a temporary workaround, try running this command inside the docker image before using auditwheel:
```
$ /opt/python/cp35-cp35m/bin/pip install -U git+https://github.com/pypa/auditwheel.git#egg=auditwheel
```
(and let us know whether it works!)

**Comment by ogrisel** [comment]
The manylinux docker images should now be up to date (that is, it includes the recently released auditwheel 1.5). Use `docker pull` to update  your local copy.

**Comment by youngpm** [comment]
Looking good!
```
+ auditwheel repair wheelhouse/python_manylinux_demo-1.0-cp27-cp27m-linux_x86_64.whl -w /io/wheelhouse/
Repairing python_manylinux_demo-1.0-cp27-cp27m-linux_x86_64.whl
Grafting: /usr/lib64/atlas/libatlas.so.3.0 -> pymanylinuxdemo/.libs/libatlas-dd66cbe8.so.3.0
Grafting: /usr/lib64/atlas/libcblas.so.3.0 -> pymanylinuxdemo/.libs/libcblas-de16e930.so.3.0
Setting RPATH: pymanylinuxdemo/extension.so to "$ORIGIN/.libs"
Previous filename tags: linux_x86_64
New filename tags: manylinux1_x86_64
Previous WHEEL info tags: cp27-cp27m-linux_x86_64
New WHEEL info tags: cp27-cp27m-manylinux1_x86_64

Fixed-up wheel written to /io/wheelhouse/python_manylinux_demo-1.0-cp27-cp27m-manylinux1_x86_64.whl
```
Thanks for figuring this out!
